src/middlewared/middlewared/plugins/vm.py

```diff
@@ -927,6 +941,11 @@ def random_mac(self):
 
         `devices` is a list of virtualized hardware to add to the newly created Virtual Machine.
         Failure to attach a device destroys the VM and any resources allocated by the VM devices.
+
+        Bhyve allows a maximum of of 16 guest virtual CPU's. By default, bhyve configures every virtual CPU as a
```

Comment:
Lets not mention bhyve, user doesnt need to know implementation details.

Also there is "of of", duplicated